image-loader/conf/routes

```diff
@@ -9,6 +9,7 @@ GET     /images/project/:imageId                      controllers.ImageLoaderCon
 # Upload Status
 GET     /uploadStatus/:imageId                        controllers.UploadStatusController.getUploadStatus(imageId: String)
 POST    /uploadStatus/:imageId                        controllers.UploadStatusController.updateUploadStatus(imageId: String)
+GET     /uploadsBy/:userId                            controllers.UploadStatusController.getUploadsBy(userId: String)
```

Comment:
I think it might be better for the server to identify the user if one isn't provided (rather than the client)
```suggestion
GET     /uploadsBy/:userId                            controllers.UploadStatusController.getUploadsBy(userId: Option[String])
```